src/components/LoginForm.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import FormControl from '@material-ui/core/FormControl';
+import Input from '@material-ui/core/Input';
+import Button from '@material-ui/core/Button';
+import {createStyles, makeStyles, Theme} from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme: Theme) => createStyles({
+  inputFields: {
+    width: '100%',
+  },
+  loginButton: {
+    width: '100%',
+  }
+}));
+
+const LoginForm = () => {
+  const classes = useStyles();
+
+  return (
+    <form>
+      <Input
+        placeholder="email"
+        type="email"
+        required
+        className={classes.inputFields}
+      />
+      <Input
+        placeholder="password"
+        type="password"
+        required
+        className={classes.inputFields}
+      />
+      <Button variant="contained" className={classes.loginButton}>Login</Button>
+    </form>
```

Comment:
I think it would be nice to have a little space (margin/padding) between textboxes.